cellprofiler/gui/pipelinecontroller.py

```diff
@@ -2709,13 +2555,9 @@ def on_stop_analysis(self, event):
                 self.__pipeline.save_measurements(path, event.measurements)
         finally:
             m = event.measurements
-            status = m[cellprofiler.measurement.IMAGE, cellprofiler.analysis.AnalysisRunner.STATUS,
-                       m.get_image_numbers()]
-            n_image_sets = sum([
-                                   x == cellprofiler.analysis.AnalysisRunner.STATUS_DONE for x in status])
+
             self.stop_running()
-            if cellprofiler.preferences.get_show_analysis_complete_dlg():
-                self.show_analysis_complete(n_image_sets)
```

Comment:
I investigated. This can be removed since the progress bar is managed elsewhere (cellprofiler.gui.preferencesview.__make_progress_panel).